### ISSUE TITLE
Build fails if ilp-kit is a dependency

### ISSUE THREAD

**Issue by dappelt** [interledgerjs/ilp-kit#158]
To get the integration tests working, the ilp-kit has to be installed as a dependency. The main difference when installing the ilp-kit as dependency is that it is installed in `node_modules`, alongside other dependencies.

This leads to problems with hard-coded paths, in which it is assumed that the containing file is not located in `module_root/node_modules/ilp-kit/src/...`, but in `module_root/src/...`. In consequence, `npm run build` fails because the path does not exist. An example for such a hard-coded path is in [src/components/HistoryItem/HistoryItem.scss](https://github.com/interledgerjs/ilp-kit/blob/master/src/components/HistoryItem/HistoryItem.scss#L3). When installing the `ilp-kit` as dependency and trying to build the `ilp-kit`, this is the error message: 
```
ERROR in ./src/components/HistoryItem/HistoryItem.scss
Module build failed: ModuleBuildError: Module build failed:
@import '../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss';
^
      File to import not found or unreadable: ../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss
Parent style sheet: stdin
      in /Users/da/git/interledgerjs/five-bells-integration-test/integration-test/node_modules/ilp-kit/src/components/HistoryItem/HistoryItem.scss (line 3, column 1)
    at DependenciesBlock.onModuleBuildFailed (/Users/da/git/interledgerjs/five-bells-integration-test/integration-test/node_modules/webpack-core/lib/NormalModuleMixin.js:315:19)
    at nextLoader (/Users/da/git/interledgerjs/five-bells-integration-test/integration-test/node_modules/webpack-core/lib/NormalModuleMixin.js:270:31)
    at /Users/da/git/interledgerjs/five-bells-integration-test/integration-test/node_modules/webpack-core/lib/NormalModuleMixin.js:292:15
    at context.callback (/Users/da/git/interledgerjs/five-bells-integration-test/integration-test/node_modules/webpack-core/lib/NormalModuleMixin.js:148:14)
    at Object.onRender (/Users/da/git/interledgerjs/five-bells-integration-test/integration-test/node_modules/sass-loader/index.js:272:13)
    at /Users/da/git/interledgerjs/five-bells-integration-test/integration-test/node_modules/sass-loader/node_modules/async/lib/async.js:906:35
    at _arrayEach (/Users/da/git/interledgerjs/five-bells-integration-test/integration-test/node_modules/sass-loader/node_modules/async/lib/async.js:85:13)
    at Object.<anonymous> (/Users/da/git/interledgerjs/five-bells-integration-test/integration-test/node_modules/sass-loader/node_modules/async/lib/async.js:898:17)
    at Object.callback (/Users/da/git/interledgerjs/five-bells-integration-test/integration-test/node_modules/sass-loader/node_modules/async/lib/async.js:44:16)
    at options.error (/Users/da/git/interledgerjs/five-bells-integration-test/integration-test/node_modules/node-sass/lib/index.js:289:32)
 @ ./src/components/HistoryItem/HistoryItem.js 75:19-48
```

When changing the problematic path from `../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss` to `../../../bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss` the build works, but obviously the build would not work anymore if the `ilp-kit` is not a dependency. Is there a way to resolve this path in a dynamic manner?

To reproduce the problem checkout [my branch of the integration tests](https://github.com/interledgerjs/five-bells-integration-test/tree/da-ilpkit-integration-tests), run `npm test`, switch into `integration-test/node_modules/ilp-kit` and try to build the `ilp-kit`.

**Comment by vhpoet** [comment]
@dappelt please try again, https://github.com/interledgerjs/ilp-kit/commit/7004ca707931448f3648e634d3deb5e4333b014b might've fixed it.

**Comment by dappelt** [comment]
Seems to be fixed by [f26a547](https://github.com/interledgerjs/ilp-kit/commit/f26a5479404e2ac18bd2b4621cfce8c24e73cc80).
